src/composables/currencies.ts

```diff
@@ -53,17 +52,19 @@ const initPollingWatcher = createPollingBasedOnMountedComponents(POLLING_INTERVA
 export function useCurrencies({
   withoutPolling = false,
   store,
-  selectedProtocol = PROTOCOL_AETERNITY, // TODO - remove default value & make the protocol required
 }: UseCurrenciesOptions) {
   const { protocolsInUse, isLoggedIn } = useAccounts({ store });
-  const minTipAmount = computed(() => 0.01 / (currencyRates.value?.[selectedProtocol].usd || 1));
-  const currentCurrencyRate = computed(
-    (): number => currencyRates.value?.[selectedProtocol]?.[currentCurrencyCode.value] || 0,
+  const aeMinTipAmount = computed(
```

Comment:
This value is used only in one place - `veeValidate` file. So I suggest to move this logic there.